labs/splice.md

```diff
@@ -1,22 +1,21 @@
----
-layout: default
-title: Splice
-parent: Labs
----
 # Splice
```

Comment:
The above content is needed so that the lab shows up on https://labs.hyperledger.org